<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterScreen</class>
 <widget class="QWidget" name="RegisterScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢迎注册</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/registerIcon.svg</normaloff>:/res/registerIcon.svg</iconset>
  </property>
  <widget class="QLabel" name="userNameLab_icon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/usernameRegister.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="passWordLab_icon">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/pwdRegister.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="surePassWordLab_icon">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/surePwdRegister.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="telephoneLab_icon">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/telephoneRegister.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="roleLab_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>480</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/roleRegister.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="emailLab_icon">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>400</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/emialRegister.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLab_tips">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>50</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:700; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="passWordLab_tips">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:700; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="surePassWordLab_tips">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>230</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:700; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="telephoneLab_tips">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>320</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:700; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="emailLab_tips">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>410</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:700; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="movieLab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>641</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MyPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>580</y>
     <width>76</width>
     <height>72</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/backRegister.svg</normaloff>:/res/backRegister.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MyPushButton" name="quitBtn">
   <property name="geometry">
    <rect>
     <x>1023</x>
     <y>580</y>
     <width>76</width>
     <height>72</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/quitRegister.svg</normaloff>:/res/quitRegister.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MyPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>580</y>
     <width>76</width>
     <height>72</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/submitRegister.svg</normaloff>:/res/submitRegister.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLab_tips_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>90</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLab">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>51</y>
     <width>121</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>用  户 名 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="passWordLab">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>140</y>
     <width>120</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>密        码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passWordEdit">
   <property name="geometry">
    <rect>
     <x>734</x>
     <y>140</y>
     <width>351</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameEdit">
   <property name="geometry">
    <rect>
     <x>734</x>
     <y>51</y>
     <width>351</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="surePassWordLab">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>229</y>
     <width>122</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>确认密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="surePassWordEdit">
   <property name="geometry">
    <rect>
     <x>734</x>
     <y>229</y>
     <width>351</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="telephoneLab">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>318</y>
     <width>120</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>电        话:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telephoneEdit">
   <property name="geometry">
    <rect>
     <x>734</x>
     <y>318</y>
     <width>351</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="emailLab">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>407</y>
     <width>127</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>邮         箱:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailEdit">
   <property name="geometry">
    <rect>
     <x>734</x>
     <y>407</y>
     <width>351</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="roleLab">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>496</y>
     <width>120</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>角        色:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="roleComboBox">
   <property name="geometry">
    <rect>
     <x>734</x>
     <y>496</y>
     <width>351</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;华文楷体&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>商务</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>技术</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>研发</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="passWordLab_tips_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>180</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="surePassWordLab_tips_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>270</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="telephoneLab_tips_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>360</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="emailLab_tips_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>450</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">mypushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
